view/theme/frio/templates/wall_thread.tpl

```diff
@@ -537,68 +537,100 @@ as the value of $top_child_total (this is done at the end of this file)
+                                                <a id="star-{{$item.id}}" href="javascript:doStar({{$item.id}});" class="btn-link {{$item.star.classdo}}" title="{{$item.star.do}}"><i class="fa fa-star-o" aria-hidden="true"></i>&nbsp;{{$item.star.do}}</a>
+                                                <a id="unstar-{{$item.id}}" href="javascript:doStar({{$item.id}});" class="btn-link {{$item.star.classundo}}" title="{{$item.star.undo}}"><i class="fa fa-star" aria-hidden="true"></i>&nbsp;{{$item.star.undo}}</a>
+                                        </li>
+                                        {{/if}}
 
-							{{if $item.follow_thread}}
-								<li role="menuitem">
-								<a id="follow_thread-{{$item.id}}" href="javascript:{{$item.follow_thread.action}}" class="btn-link" title="{{$item.follow_thread.title}}"><i class="fa fa-plus" aria-hidden="true"></i>&nbsp;{{$item.follow_thread.title}}</a>
-							</li>
-							{{/if}}
+                                        {{if $item.follow_thread}}
+                                        <li role="menuitem">
+                                                <a id="follow_thread-{{$item.id}}" href="javascript:{{$item.follow_thread.action}}" class="btn-link" title="{{$item.follow_thread.title}}"><i class="fa fa-plus" aria-hidden="true"></i>&nbsp;{{$item.follow_thread.title}}</a>
+                                        </li>
+                                        {{/if}}
 
-							{{if $item.language}}
-							<li role="menuitem">
-								<a id="language-{{$item.id}}" href="javascript:alert('{{$item.language.1}}');" class="btn-link filer-item language-icon" title="{{$item.language.0}}"><i class="fa fa-language" aria-hidden="true"></i>&nbsp;{{$item.language.0}}</a>
-							</li>
-							{{/if}}
+                                        {{if $item.language}}
+                                        <li role="menuitem">
+                                                <a id="language-{{$item.id}}" href="javascript:alert('{{$item.language.1}}');" class="btn-link filer-item language-icon" title="{{$item.language.0}}"><i class="fa fa-language" aria-hidden="true"></i>&nbsp;{{$item.language.0}}</a>
+                                        </li>
+                                        {{/if}}
 
-							{{if $item.ignore || ($item.drop && $item.drop.dropping)}}
-								<li role="separator" class="divider"></li>
-							{{/if}}
+                                        {{if $item.browsershare}}
+                                        <li role="menuitem" class="button-browser-share">
+                                                <a id="browser-share-{{$item.id}}" href="javascript:navigator.share({url: '{{$item.plink.orig}}'})" class="btn-link button-browser-share" title="{{$item.browsershare.1}}"><i class="fa fa-share-alt" aria-hidden="true"></i>&nbsp;{{$item.browsershare.0}}</a>
+                                        </li>
+                                        {{/if}}
```

Comment:
This link is included in the "Share" dropdown menu that already includes native reshare, quote share and browser share if available.
```suggestion
```

Reply:
checked it - you are right. I don´t know why intendation is messed up.
I double checked it - probably i should use another editor.

Anyway - we should first process annandos PR - mine is just copy & Paste and i am not sure if i have the rights here to merge :)
Thanky you again!